### DISCUSSION TITLE
GoPro Hero11 --- doesn't recognize QR codes

### DISCUSSION THREAD

**Discussion by CloudLabARW** [gopro/labs#1164] [General]
Greetings,

GoPro live chat suggested that I converse with you about troubleshooting. 

I'm having issues with the goPro recognizing QR codes. 

From an unopened camera box, 

1) Update the firmware using this link
https://gopro.github.io/labs/

2) Use the camera to recognize these two QR codes

https://gopro.github.io/labs/control/set/?cmd=!MTUSB=1

https://gopro.github.io/labs/control/set/?cmd=!MHDMI=1

When I try to scan the QR code, nothing happens. In previous attempts that have worked, the camera pops a message that it recognized the QR.

**Comment by dnewman-gpsw** [comment]
Does the camera report that Labs installed?

Should display this on each boot:
![image](https://github.com/user-attachments/assets/db5b14aa-773f-4f6e-88d8-94f610975ae7)

**Reply by CloudLabARW** [reply]
no, it doesn't show any message on the screen. 
No report/message/alert appears when I try to scan a QR code with the goPro.

In the past, the goPro will beep when it recognizes the QR code.

**Reply by dnewman-gpsw** [reply]
If it doesn't display the above message on boot, you haven't successfully installed Labs firmware (so it will not scan anything.)

You can also look in the about menu, as Labs version always w.xy.70 or .71 etc.  For HERO11 the about will show version 2.30.70 when you currently get Labs installed. The install instructions are here: https://gopro.github.io/labs/install/
